docker/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '3'
+
+services:
+    postgres:
+        image: postgres
+        ports:
+            - '5433:5432'
```

Comment:
There is no need to open this container to the outside, its only purpose is to be used by other services.